Sources/BrowserServicesKit/ContentBlocking/UserScripts/contentblockerrules.js

```diff
@@ -170,19 +170,21 @@
         })
     }
 
-    if (webkit.messageHandlers.processRule) {
+    if ((webkit || window.webkit).messageHandlers.processRule) {
```

Comment:
These are equivalent, I don't this will make any difference.

Reply:
I was a bit curious about this. And it actually did on my first try! (I swear I'm not making it up) but as you mentioned in the other comment the try/catch is the real change. I can remove the fallback.

Reply:
I wonder if it's just a race condition of contexts loading then?

It would be good to work out what handlers we're expecting and have enabled still. As you also mentioned perhaps some have been removed by mistake.

Reply:
In this case its not the handler that doesn't exist. Its the entire `webkit` object that is missing. On my first run the fallback to `window.webkit` fixed the issue which is documented as a divergence in iOS versions (allegedly) [here](https://stackoverflow.com/questions/64922289/cant-find-variable-webkit-on-ios-14-2). But after verifying and applying this here it no longer worked until I put in the try/catch (for obvious reasons).